### PR TITLE
some new shit butt i`m lazzy ass and i have only sprites

### PR DESCRIPTION
**Pull request by blueDolbaeb**

### CODE REVIEW

**Comment by CrAzYPiLoT-SS13** [comment]
Nice

**Comment by IlyaIU** [comment]
ЕБАНЫЙ АУТИСТ БЛЯТЬ

**Comment by Birdtalon** [comment]
Cyka Blyat!

**Comment by blueDolbaeb** [comment]
как удалить

**Comment by LightFire53** [comment]
Russians are trying to overthrow the code again...

**Comment by blueDolbaeb** [comment]
i`m autistic monkey ((

**Comment by Fruerlund** [comment]
You have my approval, even I do not know what this is. 

LGTM

**Comment by Birdtalon** [comment]
Merge immediately

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# Paradise
-
-[Website](http://www.nanotrasen.se/) - [Code](https://github.com/ParadiseSS13/Paradise) - [Discord](https://discordapp.com/invite/gXZMc?utm_source=Discord%20Widget&utm_medium=Connect)
+# ATMTA STATION
+Форк Paradise Station.
+[Website](http://атмта.рф/) - [Code](https://github.com/ParadiseSS13/Paradise) - [Discord](https://discord.gg/014REXI6YHvMSo7Ew)
 
+Постоянно модифицируемый билд Paradise, набитый мемами и МЕМАСАМИ.
```
Comment (taukausanake) [inline README.md]: Full of memes, good shit :+1:

Comment (taukausanake) [inline README.md]: I swear that was in English when I read it

File: code/game/gamemodes/changeling/changeling.dm [hunk 2]
```diff
@@ -204,10 +204,13 @@ var/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","Epsilon"
 						feedback_add_details("changeling_objective","[objective.type]|FAIL")
 						changelingwin = 0
 					count++
+					karma_reward = count - 1
 
 			if(changelingwin)
 				text += "<br><font color='green'><B>The changeling was successful!</B></font>"
 				feedback_add_details("changeling_success","SUCCESS")
+				sql_report_objective_karma(changeling.key, karma_reward)
+				to_chat(world, "<b>[changeling.key] got [karma_reward] karma points for completing objectives!</b>")
```
Comment (taukausanake) [inline code/game/gamemodes/changeling/changeling.dm]: Awarding karma for completing objectives. I'd like to see how this actually works for them

Comment (IlyaIU) [inline code/game/gamemodes/changeling/changeling.dm]: @taukausanake yeah also we tried to award karma for job objective and that went well for science dept but i can't figure out a good objectives for Sec or Engies so i just removed this feature.

**Comment by IlyaIU** [comment]
image
Пожалуйста, дыши ровно, нажми на кнопку Close pull request

File: hyntatmta/code/modules/clothing/karma/karma_clothing.dm [hunk 3]
```diff
@@ -0,0 +1,212 @@
+//Here you can see the karma outfits, which you can buy from karma shop
+//DIO's outfit
+/obj/item/clothing/head/atmta/dio
+	name = "DIO's heart headband"
+	desc = "Why is there a heart on this headband? The World may never know."
```
Comment (taukausanake) [inline hyntatmta/code/modules/clothing/karma/karma_clothing.dm]: Okay, can someone else please appreciate these Jojo references? I don't even watch it but these are gold

File: hyntatmta/code/game/objects/items/weapons/vape.dm [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+	desc = "Simple and cheap vaping device"
+	icon = 'hyntatmta/icons/obj/vape.dmi'
+	icon_state = "classic"
+	w_class = 1
+	flags = OPENCONTAINER
+	var/on = 0
+	var/lastHolder = null
+	var/smoketime = 300
+	var/chem_volume = 10
+	var/vape_cooldown = 0
```
Comment (taukausanake) [inline hyntatmta/code/game/objects/items/weapons/vape.dm]: @tigercat2000 Are you triggered? :eyes:

Comment (taukausanake) [inline hyntatmta/code/game/objects/items/weapons/vape.dm]: Also, I almost want this to be ported. Almost

Comment (taukausanake) [inline hyntatmta/code/game/objects/items/weapons/vape.dm]: nevermind, there were spaces between those before.I swear something is going on

**Pull request closed without merging**